apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -424,16 +424,46 @@ public async Task WhenSessionPoolReady()
             // End sample
         }
 
-        [Fact]
-        public void GetSessionPoolDatabaseStatistics()
+        [Fact]
+        public void GetSessionPoolStatistics()
+        {
+            string connectionString = _fixture.ConnectionString;
+
+            // Sample: GetSessionPoolDatabaseStatistics
+            // Additional: GetSessionPoolDatabaseStatistics
+            using (SpannerConnection connection = new SpannerConnection(connectionString))
+            {
+#pragma warning disable CS0618 // Type or member is obsolete
+                SessionPool.DatabaseStatistics stats = connection.GetSessionPoolDatabaseStatistics();
+#pragma warning restore CS0618 // Type or member is obsolete
+                if (stats is null)
+                {
+                    Console.WriteLine("No session pool for this connection string yet");
+                }
+                else
+                {
+                    // Access individual properties...
+                    Console.WriteLine($"Database name: {stats.DatabaseName}");
+                    Console.WriteLine($"Active sessions: {stats.ActiveSessionCount}");
+                    Console.WriteLine($"Pooled read-only sessions: {stats.ReadPoolCount}");
+                    Console.WriteLine($"Pooled read-write sessions: {stats.ReadWritePoolCount}");
+                    // ... or just use the overridden ToString method to log all the statistics in one go:
+                    Console.WriteLine(stats);
+                }
+            }
+            // End sample
+        }
+
+        [Fact]
+        public void GetSessionPoolKeyStatistics()
         {
             string connectionString = _fixture.ConnectionString;
 
             // Sample: GetSessionPoolDatabaseStatistics
```

Comment:
We might need/want to change these names. Check with me later to point you in the right direction.